pkg/lightning/errormanager/errormanager.go

```diff
@@ -764,16 +900,14 @@ func (em *ErrorManager) ReplaceConflictKeys(
 				if err2 != nil {
 					return errors.Trace(err2)
 				}
-				_, err := txn.ExecContext(c, sb.String())
+				_, err := txn.ExecContext(c, sb.String(), rowLimit)
```

Comment:
you can check the first return value `RowsAffected()`, no need to use `selectNullDataRows`